pyop2/compilation.py

```diff
@@ -185,6 +186,8 @@ def __init__(self, cc, ld=None, cppargs=[], ldargs=[],
         self._cppargs = cppargs + configuration['cflags'].split()
         if configuration["use_safe_cflags"]:
             self._cppargs += self.workaround_cflags
+        if PETSc.Log.isActive():
+            self._cppargs += ["-DPYOP2_PROFILING_ENABLED"]
```

Comment:
Is this option still required? I don't see it in the C code anywhere.

Reply:
No, it's not will remove